frontend/src/App.jsx

```diff
@@ -1,16 +1,16 @@
-import { Route, createBrowserRouter, createRoutesFromElements, RouterProvider } from 'react-router-dom'
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Homepage from './pages/Homepage';
+import PokemonDetails from './pages/PokemonDetails';
 
 function App() {
-
```

Comment:
Why change the routing ?

Reply:
I think before pushing I haven't pulled the layout added version.

Reply:
I accepted both changes and layout is now there as well.